README.md

```diff
@@ -127,33 +127,13 @@ git commit -a -m "Updated translations"
 
-## Creating custom icons
+## Icons
 
-We manage our custom font with Fontastic, so these steps can probably only be done by staff.
+We have a custom set of icons stored as SVG files and compiled into a font. New icons should be included with issues in a ready-to-use form, but some editing may be required.
 
-1. `npm install -f react-native-asset`. You'll need this to link assets.
-1. Get the new icon as an SVG file. Make sure the file name matches the icon name and no prefixes like `icon-` or `icn-`. If the icon is called "shield" the file name should be `shield.svg`.
-1. Add new icon to the iNaturalist icon set in Fontastic. **Don't change any of the existing names** and **don't change any of the existing selections** (this could mess with other projects using the font). Just add your new icon(s) to the selection.
-1. Click the "Publish" tab and click the "DOWNLOAD" button to download the zip of the font. Unzip it.
-1. Create a glyph file from the CSS file in the zip you just downloaded, using the following command (be sure to replace /path/to/styles with your path):
-    ```
-    npx generate-icon /path/to/styles.css --componentName=INatIcon --fontFamily=inaturalisticons > src/components/SharedComponents/INatIcon/INatIcon.js
-    ```
-1. When adding new icons, go to `src/components/SharedComponents/INatIcon/INatIcon.js` and make two manual changes:
-    1. First, edit the line `const iconSet = createIconSet( glyphMap, "inaturalisticons", "INatIcon.ttf" );` to `inaturalisticons.ttf` to match the name of the `.ttf` file you downloaded. There's a discrepency here because the `generate-icon` script makes an assumption that the name of the .ttf file is the same as the name of the component. Components need to start with a capital letter, and `inaturalisticons.tff` is likely not a name we want to change, since we're using it on the web too. We'll probably want to write our own `generate-icon` script to do this automatically.
-    1. Second, add the following to your exports to make sure a user can see all custom iNat icons in the UI library:
-        ```
-        export {
-          glyphMap
-        };
-        ```
-1. Add `inaturalisticons.ttf`, to `assets/fonts`.
-1. Clean build folders for iOS/Android with `rm -rf ios/build && rm -rf android/app/build`.
-1. Run `npx react-native-asset` to link the new assets.
-1. Add your icon somewhere in the app using `<INatIcon name="compass-rose" />` where name is set to the name of your new custom icon.
-1. Build the app on either platform, and you should see your custom icons.
-
-If anything goes wrong, check the [troubleshooting steps](https://github.com/oblador/react-native-vector-icons#troubleshooting) from `react-native-vector-icons` instructions on creating a glyph map.
+1. Add / edit SVGs to / in `src/images/icons/` (`git add` any new icons)
+1. `npm run icons`
+1. Rebuild the app (you'll have newly-linked assets that won't hot reload)
```

Comment:
should we add some guidance here about how to use icons?

it still bothers me a bit that we're using icons within Pressable components in some places instead of using IconButtons everywhere, but maybe that's better for a separate cleanup issue.

Reply:
I think I'd prefer a separate issue.